models/repo_list_test.go

```diff
@@ -150,7 +150,7 @@ func TestSearchRepositoryByName(t *testing.T) {
 			count: 19},
 		{name: "AllPublic/PublicAndPrivateRepositoriesOfUserIncludingCollaborative",
 			opts:  &SearchRepoOptions{Page: 1, PageSize: 10, OwnerID: 15, Private: true, AllPublic: true},
-			count: 24},
+			count: 27},
```

Comment:
```suggestion
			count: 25},
```

Two public repos were added in `models/fixtures/repository.yml`